csharp/src/Ice/StringUtil.cs

```diff
@@ -602,16 +602,12 @@ public static string UnescapeString(string s, int start, int end, string special
             return l.ToArray();
         }
 
-        public static string JoinString(string[] values, string delimiters)
+        public static string JoinStringProperty(IReadOnlyList<string> values)
         {
-            if (delimiters.Length == 0)
-            {
-                throw new ArgumentException("delimiters cannot be empty", nameof(delimiters));
-            }
-
+            string delimiters = ", \n\r\t";
             char quote = '"';
             var result = new StringBuilder();
-            char delimiter = delimiters[0];
+            char delimiter = ' ';
```

Comment:
In your previous commit this was `delimiters[0]`, which was `,`. Is this right?

Reply:
Yes, we should pick one either comma or whitespace

Reply:
My preference is comma, because it's the delimiter used when concatenating multiple options in a URI (typical example, alt-endpoint).